tests/testapps/CTCatalog/CTCatalog/Tests/CTCLineTestViewController.mm

```diff
@@ -28,8 +28,6 @@ @implementation CTTruncatedLineTestView {
 }
 
 - (void)drawRect:(CGRect)rect {
-    UIColor* color = [UIColor blueColor];
```

Comment:
didn't you already make this deletion in another commit? I could've sworn I saw this already...